### ISSUE TITLE
OpenAI API is extremely slow

### ISSUE THREAD

**Issue by JanHorcicka** [langchain-ai/langchain#11836]
### System Info

LangChain Version: 0.0.314
Python 3.9.6

I have been using LangChain with OpenAI and FAISS for building RAG chatbots. Everything was working fine until last week. 
I have noticed that for some reason, I have much higher response times. I have debugged the LangChain code and found that the reason is the LLM response itself. I am using OpenAI GPT-3.5-turbo, I haven't changed anything in the prompt being sent, but response time has increased dramatically. For example, a very simple prompt in September took 2 seconds to generate. Now the exact same prompt takes up to 14 seconds to get a response!

Does anybody have the same experience? Does anybody know why this can be happening?
Any help will be appreciated!

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [X] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

1. Use OpenAI as an LLM
2. Observer response times

### Expected behavior

The LLM response should be much faster

**Comment by JanHorcicka** [comment]
`stream=True` did not help. I tried using the `gpt-4` model and it provided a result in 14 seconds. The `gpt-3.5-turbo` took 45 seconds to provide response for the same prompt. This behaviour is persistent over different days and networks, so it's not a latency issue. 

How is it possible that the LLM is so slow?

**Comment by lekpeng** [comment]
Hi @JanHorcicka I am facing the same issue of decreased latency recently on langchain v0.0.308. Did you manage to resolve it with `gpt-3.5-turbo` or just switched to `gpt-4`?

**Comment by JanHorcicka** [comment]
@lekpeng Neither. I switched to a different LLM. I am using Palm2 by Google, but not too long ago so I cannot comment on it's quality so far.

**Comment by lekpeng** [comment]
@JanHorcicka Actually I saw many people complaining about this issue on Open AI's forum. Seems to be an ongoing issue for the past few months. Thanks for sharing the alternative you are using, will consider it.

**Comment by wshobson** [comment]
Use the `gpt-3.5-turbo-1106` model.

**Comment by Croasdell** [comment]
**@JanHorcicka** It appears many users, including myself, are encountering this sluggishness with the LLM response from the OpenAI GPT-3.5-turbo. Despite attempting the suggested remedies—like using `stream=True` and considering the token count—there's been no improvement in the response times.

This issue seems persistent across different networks and days, indicating it might not be linked to latency alone. I also experimented with the GPT-4 model, which delivered a response in a fraction of the time the GPT-3.5-turbo took, further accentuating the slowness of the latter.

While I haven't resolved this with GPT-3.5-turbo, I shifted to an alternative LLM, Palm2 by Google. It's relatively new for me, so I can't provide an extensive commentary on its performance yet.

It's concerning to note that this has been a lingering issue for a few months and seems to be echoed by multiple users on Open AI's forum. Thank you for your insights, and I'm hopeful that the alternative might offer a more efficient solution.

If anyone discovers a resolution or workaround to rectify this slowdown with the GPT-3.5-turbo, it would greatly benefit the community. Let's stay connected and share any advancements or alternative methods that could enhance the response times.

Feel free to continue this conversation here or share any new insights.
